src/main/java/com/microfocus/application/automation/tools/mc/HttpUtils.java

```diff
@@ -100,17 +99,20 @@ private static HttpResponse doHttp(ProxyInfo proxyInfo, String requestMethod, St
 
         connection.connect();
 
-
         int responseCode = connection.getResponseCode();
 
         if (responseCode == HttpURLConnection.HTTP_OK) {
             InputStream inputStream = connection.getInputStream();
             JSONObject jsonObject = convertStreamToJSONObject(inputStream);
-            Map<String, List<String>> headerFields = connection.getHeaderFields();
-            response.setHeaders(headerFields);
-            response.setJsonObject(jsonObject);
+            response.setHeaders(connection.getHeaderFields());
+            if (null == jsonObject) {
+                System.out.println(requestMethod + " " + connectionUrl + " return is null.");
```

Comment:
For me, this block will always be actived because the method `convertStreamToJSONObject` always throws a CastException. This is the case because the oauthv2 link returns a boolean value, not an object. Is this expected or not? It may interfere with the front-end login or not?